src/find-specs.js

```diff
@@ -0,0 +1,74 @@
+const fetch = require("node-fetch");
+
+const specs = require("../index.json");
+
+const nonBrowserSpecWgs = ["Accessibility Guidelines Working Group", "Accessible Platform Architectures Working Group", "Automotive Working Group", "Dataset Exchange Working Group", "Decentralized Identifier Working Group", "Distributed Tracing Working Group", "Education and Outreach Working Group", "JSON-LD Working Group", "Publishing Working Group", "Verifiable Credentials Working Group", "Web of Things Working Group"];
+const watchedBrowserCgs = ["Web Platform Incubator Community Group", "Web Assembly Community Group", "Immersive Web Community Group", "Audio Community Group", "Privacy Community Group", "GPU for the Web Community Group"];
+
+const canonicalizeGhUrl = url => (url.indexOf("github.io") > 0 && url.split("/").length === 4 ? url + '/' : url).replace('http:', 'https:');
```

Comment:
It's not too hard to interpret this, but I think something like this would be clearer:
```js
function canonicalizeGhUrl(url) {
  url = new URL(url);
  url.protocol = 'https:';
  if (url.pathname.lastIndexOf('/') === 0) {
      url.pathname += '/';
  }
  return url.toString();
}
```